config/config_test.go

```diff
@@ -25,7 +25,6 @@ func TestConfig(t *testing.T) {
 				So(cfg.HealthCheckInterval, ShouldEqual, 30*time.Second)
 				So(cfg.HealthCheckCriticalTimeout, ShouldEqual, 90*time.Second)
 				// TODO: Remove assertion once configuration variable has been removed from config.go
-				So(cfg.HelloWorldEmphasise, ShouldEqual, true)
```

Comment:
I think we should be adding assertions for the new config values here

Reply:
Yes we should, good catch.